src/seo.js

```diff
@@ -22,4 +22,4 @@ export function typeHasSeo(type) {
   return schema.get(type).fields.some(x => x.name === 'seo')
 }
 
-export const objects = [seo]
+export const objects = [seo]
```

Comment:
@coffeemedia Haal je expres de newlines weg? De conventie is om ze er wel te hebben zodat we ook niet die stopborden in Github krijgen.

Reply:
Niet on purpose. Denk dat deze verdwenen zijn vanwege een copy-paste vanuit een ander PR, icm het ontbreken van een `.editorconfig`.
Ze zijn hersteld.